internal/types/vault.go

```diff
@@ -25,11 +23,11 @@ type VaultCreateResponse struct {
 }
 
 func (v *VaultCreateResponse) Task() (*asynq.Task, error) {
-	payload, err := json.Marshal(v)
+	task, err := tasks.NewKeyGeneration("Vultisigner", v.SessionID, v.HexChainCode)
```

Comment:
I think you also need `HexEncryptionKey` in this , otherwise how the worker decrypt the message , and apply to TSS instance?